### PR TITLE
added commands for boot2docker to Makefile and updated related troubleshooting in docs

### PR DESCRIPTION
**Pull request by acsr**
new revised pullrequest.

### CODE REVIEW

**Comment by ghost** [comment]
Can one of the admins verify this patch?

File: docs/installation/development.rst [hunk 1]
```diff
@@ -100,14 +100,19 @@ so you can easily edit them with your favorite IDE.
 3. `cd ploneintranet`
 4. `sudo make docker-build` will prepare the docker image
 5. `sudo make docker-run` will launch the docker image and give you a shell inside
-6. `make` will fetch the ploneintranet compiled javascript bundle
-7. `make devel` will run the development buildout
-8. `bin/instance fg` will run the Plone instance, which you can access on localhost:8080.
+
+.. note:: Make sure you are inside the shell (usually your prompt changes!)
```
Comment (gyst) [inline docs/installation/development.rst]: "shell" is not the right word here. All these commands are shell commands. Should be "inside the docker container".

File: Makefile [hunk 2]
```diff
@@ -98,16 +98,42 @@ jsrelease: prototype
...
+# start without sudo: 'make boot2docker-run'
+boot2docker-build: .ssh/known_hosts
+	docker build -t $(PROJECT) .
+
 # re-uses ssh agent
 # also loads your standard .bashrc
 docker-run:
-	docker.io run -i -t \
+	docker run -i -t \
+                --net=host \
```
Comment (gyst) [inline Makefile:117]: Please don't change this from "docker.io" to "docker". It will break on Ubuntu.

**Comment by gyst** [comment]
@acsr You did not respond to my comments.

**Comment by acsr** [comment]
@gyst I actually missed the comments because I was new to the waffle view. Changes done! I now move it again. Do not slap me, if I miss something again to complete the PR.

File: Makefile [hunk 3]
```diff
@@ -153,7 +179,6 @@ solr-clean:
 # inspect robot traceback:
 # bin/robot-server ploneintranet.suite.testing.PLONEINTRANET_SUITE_ROBOT
 # firefox localhost:55001/plone
-# To see the tests going on, use DISPLAY=:0, or use Xephyr -screen 1024x768 instead of Xvfb
```
Comment (gyst) [inline Makefile:156]: Please keep this comment, I use that.

**Comment by acsr** [comment]
Now the diff to the latest commit 06cd9f6 by @ale-rt seems to look ok. Any hints to check the diffs better in the future? Or is this normal drift?

**Comment by gyst** [comment]
@acsr Thanks for this pull. If you have another one, please use a branch on ploneintranet instead of a fork in your own repo. That way it's easier to test-drive changes. You have full code access.

**Pull request merged**